src/services/photo/index.js

```diff
@@ -1,13 +1,13 @@
 const errors = require('feathers-errors');
-const GCSUploader = require('../../utils/gcs-uploader');
+const Promise = require('bluebird');
```

Comment:
The dependencies from npm will be put on top and the internal file dependencies come after. Then, we sort alphabetically in these groups.